compiler/modules/front.syntax/parseTypeDesc.bal

```diff
@@ -277,27 +277,35 @@ function parseTypeParam(Tokenizer tok) returns TypeDesc|err:Syntax {
 }
 
 // current token should be "("
-function parseFunctionTypeDesc(Tokenizer tok, string[]? paramNames = ()) returns FunctionTypeDesc|err:Syntax {
+function parseFunctionTypeDesc(Tokenizer tok, FunctionParam[]? fnParams = ()) returns FunctionTypeDesc|err:Syntax {
     // skip "function"
     Position startPos = tok.currentStartPos();
     check tok.expect("(");
-    TypeDesc[] args = [];
+    FunctionTypeParam[] params = [];
```

Comment:
Use typeParams for this variable.

Reply:
variable removed in favor of function parameter